### ISSUE TITLE
localization / i18n

### ISSUE THREAD

**Issue by darrellanderson** [darrellanderson/TI4-TTS#26]
Move mod-detected "names" to the new TTS tag feature, use TTS localized names instead.

Would require translations for all names AND images for full support (be aware even a single translation would probably increase mod size above the non-host save size limit).

**Comment by Somberlord** [comment]
A simple i18n object would be sufficient (for all displayed text), with English as default for all non-translated text.

That way, it is easy do do another mod with full image support, but keeping up with this mod's devs. (That mod exists for French, but we have to change every text in the scripts, then report updates manually)

**Comment by darrellanderson** [comment]
Yep, once all the other scripts stop using the (en) object names to detect things there are a few ways to embrace i18n.

TTS does support mixed localizations so for instance French and English players can be in a shared game but see different text *and* images.  The downside is images are already quite verbose, the mod save size would get considerably larger to have a second (or more) language integrated.

Moving text to a helper is reasonable approach if having a blanket text swap is acceptable, potentially even reading the translations from a URL or google sheet.

**Comment by darrellanderson** [comment]
Object.memo : A string where you may persist user-data associated with the object. Tabletop Simulator saves this field, but otherwise does not use it. Store whatever information you see fit.
